### DISCUSSION TITLE
Thermalisation period in training MD data

### DISCUSSION THREAD

**Discussion by romankempt** [deepmodeling/deepmd-kit#1997] [Q&A] [answered]
Sorry for this very simple question! In MD simulations, there is usually a Thermalisation period (or annealing) which is discarded before one postprocesses the data (e.g., for pressure analysis, mean temperature, etc.) Depending on the system and pre-thermalisation, this can be a few hundred steps or 2-5 pico seconds.

Should this data also be discarded for deepmd training purposes or is it safe to include it?

**Comment by wanghan-iapcm** [comment, marked as answer]
The DP model should be able to thermalise the system, thus including the thermalising part of the AIMD trajectory would be helpful.  

The capacity of the DP model with default parameters is large enough so you can safely include the thermalising data without loss of accuracy.

**Reply by romankempt** [reply]
Thank you very much!
